database/Seeders/eggs/source-engine/egg-ark--survival-evolved.json

```diff
@@ -4,29 +4,39 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-03-04T06:38:31+02:00",
+    "exported_at": "2021-06-05T15:47:39-04:00",
     "name": "Ark: Survival Evolved",
-    "author": "dev@shepper.fr",
+    "author": "support@pterodactyl.io",
```

Comment:
Why remove the credit of the original author?

Reply:
That's what was in the egg when I exported it, as we found out I had an old version of the egg. Will be updated.